docs/en/advanced_guides/datasets.md

```diff
@@ -185,55 +59,27 @@ Then in the config, to use `Filelist` you can modify the config as the following
 ```python
 train = dict(
     type='Filelist',
-    ann_file='image_list.txt',
-    pipeline=train_pipeline
+    ann_file = 'image_list.txt',
+    pipeline=transfrom_list
 )
 ```
 
-## Use dataset wrapper
-
-The dataset wrapper is a kind of class to change the behavior of dataset class, such as repeat the dataset or
-re-balance the samples of different categories.
-
-### Repeat dataset
+### Import Dataset Class (MMCls as 3rd-party package)
```

Comment:
This part can be removed. If users use MMClassification as a 3rd-party package, they probably won't use our config system. Developing a new repository based on our config system and using MMClassification as a 3rd-party package should be in another senior tutorial, instead of here.